wallets/metamask/package.json

```diff
@@ -15,17 +21,18 @@
   ],
   "scripts": {
     "build": "pnpm run clean && pnpm run build:dist && pnpm run build:types",
-    "build:cache": "synpress-cache test/wallet-setup",
-    "build:cache:headless": "synpress-cache test/wallet-setup --headless",
-    "build:cache:headless:force": "synpress-cache test/wallet-setup --headless --force",
+    "build:cache": "synpress-cache test/playwright/wallet-setup",
```

Comment:
`build:playwright:cache`?

Reply:
end goal will be to use it for both: Playwright and Cypress